tests/phpunit/api/v3/MembershipTypeTest.php

```diff
@@ -65,12 +65,12 @@ public function testGet() {
       'id' => $id,
     );
     $membershiptype = $this->callAPIAndDocument('membership_type', 'get', $params, __FUNCTION__, __FILE__);
-    $this->assertEquals($membershiptype['values'][$id]['name'], 'General', 'In line ' . __LINE__ . " id is " . $id);
+    $this->assertEquals($membershiptype['values'][$id]['name'], 'General' . " id is " . $id);
```

Comment:
This is what's causing the test failure... need to remove the extra stuff that was being concatenated on the end.